draft-rieckers-emu-eap-ute.md

```diff
@@ -390,7 +394,7 @@ As in the Waiting Exchange, the peer probes the server with a Client Completion
 The nonce of the previous Client Completion Requests which did not lead to a completion MAY be repeated.
 If the peer has received an OOB message, the peer will include the OOB-Id in the Completion Request.
 If the peer did not include an OOB-Id, the server will include the OOB-Id of its received OOB message.
-In the unlikely case, that both directions are negotiated and an OOB message is delivered from the peer to the server and from the server to the peer at the same time, as a tiebreaker the OOB message from the server to the peer is chosen.
+In the unlikely case that both directions are negotiated and an OOB message is delivered from the peer to the server and from the server to the peer at the same time, the OOB message from the server to the peer is chosen as a tiebreaker.
```

Comment:
I'd leave the "as a tiebreaker" there (see RFC9140, section 3.2.4, paragraph 6), to put an emphasis on the tiebreaker situation.